src/docs/components/site-header.svelte

```diff
@@ -4,20 +4,22 @@
 	import { navConfig, siteConfig } from '$docs/config.js';
 	import { cn } from '$docs/utils';
 	import Search from '$routes/(landing-ui)/search.svelte';
+	import { mode } from 'mode-watcher';
 	import Logo from './logo.svelte';
+	import ThemeSwitch from './nav/theme-switch';
 
 	$: isRoot = $page.url.pathname === '/';
 </script>
 
 <div
 	class={cn(
-		'container flex h-14 items-center rounded-md bg-neutral-800',
-		isRoot && 'md:bg-transparent'
+		'container flex h-14 items-center rounded-md bg-neutral-800 text-white !outline outline-2 !outline-neutral-700 dark:!outline-none',
+		isRoot && '!outline-0 md:bg-transparent'
 	)}
 >
 	<div class="flex">
 		<a href="/" class="mr-6 flex items-center transition-opacity hover:opacity-75">
-			<Logo class="h-9" withText textColor="white" />
+			<Logo class="h-9" withText textColor={$mode === 'light' ? 'black' : 'white'} />
```

Comment:
sorry for being so late to comment on this, I was busy with IRL yesterday.

for FOUC prevention. this works for dark mode users, but I'm not sure it will prevent FOUC for light mode users.

I think the best way in general to do stuff like this that can cause FOUC is to not rely on stores (since they load later than the FOUC prevention snippet) and instead rely on presence of the `dark` class.

for example:
```svelte
<Logo class="h-9 text-black block dark:hidden" withText />
<Logo class="h-9 text-white hidden dark:block" withText />
```

you can use `$mode` and the other stores to interact with the theme switching but to fully avoid FOUC it's not great.

I think the current implementation above will cause FOUC if the user has selected light mode and they do a full refresh.